public/js/game.js

```diff
@@ -114,6 +128,8 @@ interact('.play-color').dropzone({
       event.relatedTarget.classList.add('hidden');
       event.target.setAttribute('data-target', false)
       event.relatedTarget.setAttribute('data-target', false)
+      clearInterval(scoreTimer);
+      clearInterval(timeTimer);
       setTimeout(() => {
         level++;
         if (level > 4) {
```

Comment:
So it tries to get level data if the level is above 4, but we don't actually have any more than 4 levels. Should this be <= 4? Or am I missing something?

Reply:
the next line says to console.log game over.  I haven't handled that yet.